.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: run-acceptance-test
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    pipelinesascode.tekton.dev/pipeline: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/pipelines/acceptance-tests.yaml"
+    pipelinesascode.tekton.dev/on-target-branch: "[master]"
+    pipelinesascode.tekton.dev/task-01: "git-clone"
+    pipelinesascode.tekton.dev/task: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cleanup-cloud-resources.yaml"				
+    pipelinesascode.tekton.dev/task-1: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-gitlab-mr-test-results-to-dashboard.yaml"
+    pipelinesascode.tekton.dev/task-2: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cli-tests-in-container.yaml"				
+    pipelinesascode.tekton.dev/task-3: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-interop-notification.yaml"
+    pipelinesascode.tekton.dev/task-4: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cli-tests.yaml"						
+    pipelinesascode.tekton.dev/task-5: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-results-to-dashboard.yaml"
+    pipelinesascode.tekton.dev/task-6: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/clone-catalog-source.yaml"				
+    pipelinesascode.tekton.dev/task-7: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-slack-notification-generic.yaml"
+    pipelinesascode.tekton.dev/task-8: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/configure-operator-for-unofficial-builds.yaml"		
+    pipelinesascode.tekton.dev/task-9: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-slack-notification.yaml"
+    pipelinesascode.tekton.dev/task-10: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/configure-operator.yaml"					
+    pipelinesascode.tekton.dev/task-11: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/setup-testing-accounts.yaml"
+    pipelinesascode.tekton.dev/task-12: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/create-rhel-vm-and-subscribe.yaml"			
+    pipelinesascode.tekton.dev/task-13: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/ui-tests.yaml"
+    pipelinesascode.tekton.dev/task-14: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/create-rp-launch.yaml"					
+    pipelinesascode.tekton.dev/task-15: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/uninstall-cluster.yaml"
+    pipelinesascode.tekton.dev/task-16: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/entitlement-tests.yaml"					
+    pipelinesascode.tekton.dev/task-17: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/unsubscribe-and-destroy-rhel-vm.yaml"
```

Comment:
You don't need to include all tasks, only those used in `acceptance-tests` pipeline

Reply:
Still not addressed, it would also make sense to sort these somehow so that it's easier to maintain if the pipeline changes (added/removed tasks etc). Alphabetically is probably the easiest approach